src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -19,11 +19,24 @@
 
     @Override
     public CinemaHall get(Long id) {
-        return cinemaHallDao.get(id).get();
+        return cinemaHallDao.get(id).orElseThrow(
+                () -> new RuntimeException("cinemaHall not found"));
     }
 
     @Override
     public List<CinemaHall> getAll() {
         return cinemaHallDao.getAll();
     }
+
+    @Override
+    public Boolean update(CinemaHall cinemaHall) {
+        CinemaHall currentCinemaHall = get(cinemaHall.getId());
+        cinemaHallDao.update(cinemaHall);
+        return currentCinemaHall.equals(get(cinemaHall.getId()));
```

Comment:
The logic in the `update` method is inefficient. Instead of comparing the current and updated `CinemaHall` objects, consider returning `true` after a successful update operation.